Library/Formula/bazel.rb

```diff
@@ -20,6 +20,8 @@ def install
       "/etc/bazel.bazelrc",
       "#{etc}/bazel/bazel.bazelrc"
 
+    ENV["EMBED_LABEL"] = "#{version}-homebrew"
```

Comment:
The build script extract the version from the environment. Our CI set the version depending on the git tag. The compile.sh script knows nothing about the version label.